apps/events.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: docs
+key: events
+title: Events
+---
+
+Events are triggered when a specific action is done.
+So, if you implement one of these events, your app will be informed about related actions and can react accordingly.
+This can be helpful if your app needs information like the current page the user is on, or the fact that a product was added to the cart.
+
+These events are currently available:
+
+| Event | Type | Description |
+| - | - |  - |
+| pageview | string | A page is rendered. Gives information on the path of the page. |
+| product | immutable.js object | A product page is rendered. Gives information on the respective product. |
+| category | immutable.js object |A category page is rendered. Gives information on the respective category and related products.|
+| cart:add | immutable.js object | A product was added to the cart. Gives information on the current state of the cart, e.g. included items. This event isn't triggered if the customer makes changes in the cart itself, e.g. changing the amount of a product. |
+
+## Pageview event
+
+In order to make use of this event, you need to add the following snippet to your code:
+
+```js
+if (window.eComEventTarget) {
+  window.eComEventTarget.addEventListener('pageview', function (event) {
+    console.log('pageview url:', event.detail.url)
+  })
+}
+```
+
+These are the information you'll get with this event:
+
+```
+pageview url: /about-us
+```
+
+## Product event
```

Comment:
Please check, what it looks like, if you put the event itself in the headline also in backticks.